apps/system/utils/utils_ps.c

```diff
@@ -65,7 +65,7 @@
 
 #include "utils_proc.h"
 
-#define PS_BUFLEN 64
+#define PS_BUFLEN (64 + CONFIG_TASK_NAME_SIZE)
 
```

Comment:
why to add CONFIG_TASK_NAME_SIZE length? 
do you worry about array overflow?
if so, i think you can define like 
 PS_BUFLEN  =  fix_length + variable_len(CONFIG_TASK_NAME_SIZE)
and pls check if fix_lenght is 64b or not.

Reply:
Please refer details I wrote in commit description.

Reply:
okay, so except long task name, how many bytes need for remain? 
I mean whether can decrease 64b, although not save much money.

Reply:
```c
/* Determines the size of an intermediate buffer that must be large enough
 * to handle the longest line generated by this logic.
 */
#define STATUS_LINELEN 64
```
This is the buffer length defined in procfs module, so we use same 64 bytes and add task name size here.